src/components/info-container.tsx

```diff
@@ -1,6 +1,10 @@
+import { useState } from "react";
 import { Text, View, StyleSheet } from "react-native";
 import { LocationObject } from "expo-location";
-import { LatLng } from "react-native-maps";
+// import * as Location from "expo-location";
+// import * as Cellular from "expo-cellular";
```

Comment:
These commented-out imports are probably best left deleted; we can use version control to get them back if we need to.